samples/drumthumper/src/main/cpp/DrumPlayerJNI.cpp

```diff
@@ -135,7 +135,7 @@ JNIEXPORT void JNICALL Java_com_plausiblesoftware_drumthumper_DrumPlayer_clearOu
  */
 JNIEXPORT void JNICALL Java_com_plausiblesoftware_drumthumper_DrumPlayer_restartStream(JNIEnv*, jobject) {
     sDTPlayer.resetAll();
-    if (sDTPlayer.openStream() && sDTPlayer.startStream()){
+    if (sDTPlayer.openStream(oboe::PerformanceMode::LowLatency) && sDTPlayer.startStream()){
```

Comment:
This seems not needed given there is a default value.

Reply:
Done